README.md

```diff
@@ -8,20 +8,20 @@ See the LICENSE and NOTICE files that should have been provided along with this
 
 ### Project Status: alpha
 
-The project is currently alpha which means that there are still new feautres
+The project is currently alpha which means that there are still new features
 and APIs planned that will be added in the future.
 Due to this, breaking changes may still happen.
 
 ## Overview
 
-This project contains the APIcast operator software. APIcast operator is a
+This project contains the APIcast operator software. APIcast operator is a piece of
 software based on [Kubernetes operators](https://coreos.com/operators/) that
 provides:
-* A way to install a 3scale APIcast self-managed solution, providing configurability
-  options at the time of installation
+* An easy way to install a 3scale APIcast self-managed solution, providing configurability
+  options at the time of installation.
 
-This functionalities definitions are provided via Kubernetes custom resources
-that the operator is able to understand and process.
+The functionalities and definitions are provided via Kubernetes custom resources
```

Comment:
I'm not sure about this @miguelsorianod as i don't know what you want to say

Reply:
It's ok. Here we meant that the functionalities are provided via Kubernetes CRs that the operator can understand :+1: